stable/nats/templates/tls-secret.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .name }}
+  labels:
+    app: "{{ template "nats.name" $ }}"
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
```

Comment:
use the template: `chart: "{{ template "nats.chart" . }}"`

Reply:
👍